### DISCUSSION TITLE
目录字体设置

### DISCUSSION THREAD

**Discussion by wangyizhuo777** [stone-zeng/fduthesis#328] [Q&A] [answered]
生成目录中子标题如何修改格式呢，想统一格式不要楷体

**Comment by sikouhjw** [comment]
如果是格式检查的要求，请注明。如果不是，只是「想」，那改动的意义及动力不大。

**Reply by wangyizhuo777** [reply]
导师的要求。所以请问可以指个路吗，在配置的哪里改动可以？

**Reply by sikouhjw** [reply, marked as answer]
https://github.com/stone-zeng/fduthesis/blob/d84e0dd7df657d47943e98c98ff843cc1336bc5f/source/fduthesis.dtx#L5338-L5343

**Reply by wangyizhuo777** [reply]
感谢~
